.github/workflows/expense.yml

```diff
@@ -41,31 +42,23 @@ jobs:
     steps:
       - uses: octokit/request-action@v2.1.9
         with:
-          route: GET /orgs/:organisation/teams/:team/memberships/${{
-            github.actor }}
+          route: GET /orgs/:organisation/teams/:team/memberships/${{ github.actor }}
           team: technical-steering-committee
           organisation: webdriverio
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_WDIO_BOT }}
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
**visual-testing** is using different tokens than **webdriverio**. Is that expected?

Reply:
AFAIK yes, @christian-bromann implemented the flow under some "pressure" so it might be that it didn't follow "the standards". But he can confirm that the best 😅

Reply:
Oh my. Any chance you've got a way to get his attention so he reviews this one first before any others in the org?

For the **webdriverio** org I copy/pasted from **webdriverio/webdriverio**:
https://github.com/webdriverio/webdriverio/blob/main/.github/workflows/expense.yml#L49

I'm happy to push the change to all 27 repos. I've got some practice with the `gh` CLI tool now. :relaxed:

Reply:
Let's keep `GH_TOKEN_WDIO_BOT` for the `webdriverio` org. I created it for this purpose. The `webdriverio/webdriverio` repo uses `GH_TOKEN` which is 4 years old and I would like to remove it some time.

Reply:
OK. I'll push that change out in **expense.yml**. Want me to clean up the remaining 5 uses at the same time?

```
```

Reply:
Yeah, that would be helpful. The `GH_TOKEN` I think is from me and it is better to use a bot token everywhere.